repo_health/index/static/karma.conf.js

```diff
@@ -27,7 +27,7 @@ module.exports = (config) => {
     ],
     plugins: [
       require('karma-jasmine'),
-      require('karma-chrome-launcher'),
+      require('karma-phantomjs-launcher'),
```

Comment:
I thought phantomjs didn't work with some of the functionality we have. Also, the code should be transpiled before the tests run, so maybe we just need to look into that. Maybe babel isn't correctly synthesizing the code correctly 😕 .